ui/apps/akasha/src/extensions/inline-editor/post-editor/index.tsx

```diff
@@ -159,10 +159,10 @@ export function PostEditor({ appName, postId, pubKey, singleSpa, action, draftSt
       setEditorState={(value: IEntryData['slateContent']) => {
         if (!canSaveDraft) return;
         if (isEqual(value, editorDefaultValue)) {
-          draft.clear({ pubKey, appName });
+          draft.clear();
           return;
         }
-        draft.save({ pubKey, appName, content: value });
+        draft.save();
```

Comment:
you still need to pass the content to be saved here

Reply:
Ouch! I just changed the constructor function into a class to get solid typing.